src/Adapter/StockManager.php

```diff
@@ -92,16 +92,24 @@ public function isAsmGloballyActivated()
      * @param $shopId
      * @param $errorState
      * @param $cancellationState
+     * @param bool $idProduct
+     * @param bool $idOrder
      * @return bool
      */
-    public function updatePhysicalProductQuantity($shopId, $errorState, $cancellationState)
+    public function updatePhysicalProductQuantity($shopId, $errorState, $cancellationState, $idProduct = false, $idOrder = false)
```

Comment:
I'd prefer `$idProduct` and `$idOrder` to be `int|null` instead of `int|false`. It will save us headaches in the long term, trust me...